packages/ethereum/contracts/HoprChannels.sol

```diff
@@ -24,46 +25,34 @@ contract HoprChannels is IERC777Recipient, ERC1820Implementer {
     // used by {tokensReceived} to distinguish which function to call after tokens are sent
     uint256 public FUND_CHANNEL_MULTI_SIZE = abi.encode(false, address(0), address(0), uint256(0), uint256(0)).length;
```

Comment:
'FUND_CHANNEL_SIZE' and 'FUND_CHANNEL_MULTI_SIZE' => `shouldOpen` no longer needed, first paramater